src/Main.elm

```diff
@@ -1,10 +1,251 @@
 module Main exposing (main)
 
-import Data.Audience
-import Data.AudienceFolder
-import Html exposing (Html)
+{-| Using elm-live for hot reloading to make development just a bit more comfortable. Overall prefer using create-elm-app because of it's debugging features,
```

Comment:
I don't recommend you `create-elm-app` anymore, it is unmaintained and it does not work in Apple M1 machines, if you google a bit you'll find parcel 2 and vite templates that do exactly the same and are fully customisable 😉

Reply:
Didn't know there was an alternative, will definitely take a look!